test-case/verify-tplg-binary.sh

```diff
@@ -44,7 +44,7 @@ main()
     # This one can find more problems, see sof-test#1054
     dlogi "Checking topology file with tplgtool2.py: $tplg_path"
     ( set -x
-      tplgtool2.py -D /tmp/ "$tplg_path" ) || {
```

Comment:
do we need to update CI test ?

Reply:
Thanks @RanderWang , can you clarify which test?

This line was added only last week so I can't imagine something already depending on it.

Maybe you're referring to the graph that is appearing in test results? That is elsewhere, much much older and unrelated.

Reply:
OK, I get your idea. I first considered some CI test may use this path